docs/pricing-structures.md

```diff
@@ -0,0 +1,39 @@
+## Pricing Structures - For Cypherpunktimes.com Proposal 2024-25
+This document details the pricing structures for each participant in the cypherpunktimes.com proposal. We have now removed the editor role, art and technical articles from this proposal. This will help standardise payments across the proposal and provide greater transparency. 
+
+There remains opportunities for creating advanced content like technical articles and art, but as these tend to be specialist in nature, it makes sense to provide separate proposals for this content.
+
+## Article pricing
+
+As detailed in the process document (https://github.com/PhoenixGreen/Cypherpunk-Times/blob/main/docs/process.md) cypherpunktimes.com has now moved to a fixed payment model for producing paid for articles, which consist of four independent stages and one optional stage.
+
+The aim of this structure is to incentivise high quality written content, whilst standardising author payments. Writers are now incentivised to participate in planning, discussion, peer review, and social media engagement.
+
+As part of this process, and to allow for more time on each stage, we are limiting each writer to a maximum of two articles per month. 
+
+An approved article now has the associated stages and hourly rate applied to it:
+
+Stage 1: Discussion and planning = 1hr
+Stage 2: Create article = 5hrs
+Stage 3: Review = 1hr
+Stage 4: Social media = 1hr
+Optional Stage: TwitterSpace / Livestream = 3hrs
+
+## Official reviewer pricing
+
+As detailed in the process document (https://github.com/PhoenixGreen/Cypherpunk-Times/blob/main/docs/process.md) reviewers are now responsible for doing quality checks and giving constructive feedback for possible improvements. The aim of the review process is that each article will have one or more reviewers conduct planning, pre-review and review checks.
+
+Maximum payment for planning and pre-review checks = 1 to 2hr
+Maximum payment for article review checks = 2 to 4hrs
+
+## Platform admin pricing
+
+The cypherpunktimes.com admin is responsible for: 
+* Maintaining the publication
+* Article Pre-flight checks
+* Publishing finished articles
```

Comment:
If I understand the process correctly we need add these too:

- Curating the process/policy documents (including this document)
- Operating the cypherpunktimes Twitter account (announcing new articles etc.)

Maybe also clarify the scope of "pre-flight" checks, it reads like it is not including thorough proofreading/reviews (which makes sense since they are covered separately in the above section).

Reply:
Added these points